_api-reference/cat/cat-segment-replication.md

```diff
@@ -39,10 +39,11 @@ Parameter | Data type | Description
 `active_only` | Boolean | If `true`, the response only includes active segment replications. Defaults to `false`. 
 [`detailed`](#additional-detailed-response-metrics) | String | If `true`, the response includes additional metrics for each stage of a segment replication event. Defaults to `false`.
 `shards` | String | A comma-separated list of shards to display.
+`byte` | Byte size | [Units]({{site.url}}{{site.baseurl}}/opensearch/units/) used to display byte size values.
 `format` | String | A short version of the HTTP accept header. Valid values include `JSON` and `YAML`.  
 `h` | String | A comma-separated list of column names to display. 
 `help` | Boolean | If `true`, the response includes help information. Defaults to `false`.
-`time` | Time value | [Units]({{site.url}}{{site.baseurl}}/opensearch/units) used to display time values. Defaults to `ms` (milliseconds).
+`time` | Time | [Units]({{site.url}}{{site.baseurl}}/opensearch/units/) used to display time values.
```

Comment:
Same in this line... if `time` is not specified, what are the units used to display time values?
```suggestion
`time` | Time units | [Units]({{site.url}}{{site.baseurl}}/opensearch/units/) used to display time values.
```

Reply:
Please see my above comment https://github.com/opensearch-project/documentation-website/pull/4731#discussion_r1289419093. Forgot to add a clear example for the default situation earlier.